src/libs/actions/OnyxUpdates.ts

```diff
@@ -65,7 +65,7 @@ function apply({lastUpdateID, type, request, response, updates}: OnyxUpdatesFrom
         console.debug('[OnyxUpdateManager] Update received was older than current state, returning without applying the updates');
         return Promise.resolve();
     }
-    if (lastUpdateID && lastUpdateIDAppliedToClient && Number(lastUpdateID) > lastUpdateIDAppliedToClient) {
+    if (lastUpdateID && (lastUpdateIDAppliedToClient == null || Number(lastUpdateID) > lastUpdateIDAppliedToClient)) {
```

Comment:
is this intentionally not ===?

Reply:
null has no type, so I didn't care for it. I can change it